patroni/scripts/wale_restore.py

```diff
@@ -83,16 +104,16 @@ def should_use_s3_to_create_replica(self):
 
             backup_info = dict(zip(names, vals))
         except subprocess.CalledProcessError as e:
-            logger.error("could not query wal-e latest backup: {}".format(e))
-            return False
+            logger.exception("could not query wal-e latest backup: {}".format(e))
```

Comment:
It should be just `logger.exception("could not query wal-e latest backup')` because `logger.exception` will print stacktrace